tests/desktop/test_collections.py

```diff
@@ -44,3 +44,28 @@ def test_user_my_collections_page(self, base_url, selenium, logged_in, user):
         my_collections_page = home_page.header.click_my_collections()
         assert 'Collections by %s :: Add-ons for Firefox' % user['display_name'] == my_collections_page.page_title
         assert 'Collections by %s' % user['display_name'] == my_collections_page.my_collections_header_text
+
+    @pytest.mark.native
+    @pytest.mark.xfail(
+        'addons-dev' in pytest.config.getvalue('base_url'),
+        reason='Problem with updating state on addons-dev')
```

Comment:
We should reference the issue in addons-server, or perhaps raise a new issue as this is similar to https://github.com/mozilla/addons-server/issues/2442 but may be a different bug.